rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/scheduler/MultiChannelSchedulerDAO.java

```diff
@@ -183,12 +183,12 @@ int scheduleJob(
 
     @Mapper(TimeJobInfoListMapper.class)
     @Query(
-            "local expiredJobs = redis.call('ZRANGEBYSCORE', $queue$, 0, $expirationTimeInMillis$, 'WITHSCORES')\n" +
-            "redis.call('ZREMRANGEBYSCORE', $queue$, 0, $expirationTimeInMillis$)\n" +
+            "local expiredJobs = redis.call('ZRANGEBYSCORE', $runningQueue$, 0, $expirationTimeInMillis$, 'WITHSCORES')\n" +
+            "redis.call('ZREMRANGEBYSCORE', $runningQueue$, 0, $expirationTimeInMillis$)\n" +
             "return expiredJobs"
     )
-    List<TimeJobInfo> removeExpiredJobs(@BindKey("queue") String queue,
-                                        @BindArg("expirationTimeInMillis") Long expirationTimeInMillis);
+    List<TimeJobInfo> removeExpiredRunningJobs(@BindKey("runningQueue") String runningQueue,
+                                               @BindArg("expirationTimeInMillis") Long expirationTimeInMillis);
```

Comment:
wait.. how are we removing expired ready jobs after this change?

Reply:
we were already doing that only through removeAllExpiredReadyJobs. changing this just makes it clearer that this is the purpose of the method.